packages/react/src/components/breadcrumbs/Breadcrumb.tsx

```diff
@@ -5,23 +5,27 @@ import {ITitleProps, Title} from '../title/Title';
 import {BreadcrumbLink, IBreadcrumbLinkProps} from './BreadcrumbLink';
 
 export interface IBreadcrumbProps extends React.ClassAttributes<Breadcrumb> {
+    /**
+     * Array of breadcrumb links to navigate to previous pages
+     */
     links?: IBreadcrumbLinkProps[];
-    defaultLinkPath?: string;
+    /**
+     * The content of the current breadcrumb

```

Comment:
Just to make it clear it's the title text, although maybe it being breadcrumb props is enough.

```suggestion
     * The title of the current breadcrumb
```

Reply:
I think the name is misleading because `title` here contains the title (string) + an optional svgLink :thinking: 
So not just the title :sweat_smile:

Reply:
Makes sense 👍